extensions/docs/msal-node-extensions.md

```diff
@@ -11,17 +11,49 @@ Supported platforms are Windows, Mac and Linux:
 - Linux - LibSecret is used for storing to "Secret Service" through npm keytar.
 
 ## Code
+
 ### Creating the persistence layer
-Creating the persistence will differ based on what platform you are targeting.
 
+API for creating the persistence layer will differ based on what platform you are targeting.
+
+Alternatively, you can use PersistenceCreator.createPersistence as it's a generic wrapper and selects the appropriate persistence based on the platform . 
+
+```js
+const { protectedResources, msalConfig, persistenceConfiguration } = require("./authConfig");
+const {
+  DataProtectionScope,
+  PersistenceCreator,
+  PersistenceCachePlugin,
+} = require("@azure/msal-node-extensions");
+
+let authProvider;
+authProvider = new AuthProvider(msalConfig);
+...
+
+PersistenceCreator
+        .createPersistence(persistenceConfiguration)
+        .then((persistence) => {
```

Comment:
Should we change this to: `const persistence = await PersistenceCreator.createPersistence(persistenceConfiguration)`?

Should we be using await instead of .then here and everywhere else in this file?

Reply:
+1